assets/airports/kmsy.json

```diff
@@ -0,0 +1,2833 @@
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "airways": {
+        "A770": ["LEV", "DOLPH", "ALGAE"],
+        "J2"  : ["LCH", "LSU", "SJI", "MUURY", "INBRD", "CEW", "DEFUN"],
+        "J31" : ["LEV", "HRV", "MEI"],
+        "J58" : ["HRV", "PEBBY", "AEX"],
+        "L214": ["LEV", "PLNDR", "DAGGR"],
+        "L333": ["HRV", "HOOCK", "TRESR"],
+        "M345": ["TBD", "WAHOO", "KENGS"],
+        "Q22" : ["OYSTY", "ACMES", "CATLN", "TWOUP", "SPA"],
+        "Q24" : ["LSU", "IRUBE", "PAYTN"],
+        "Q56" : ["PEKON", "HRV", "SJI", "CATLN"],
+        "Q105": ["HRV", "FATSO", "REDFN"],
+        "V9"  : ["LEV", "PEPSI", "GOING", "SAFES", "RAYOP", "WAVEZ", "OYSTY", "RYTHM", "FOILS", "ZYDCO", "MCB"],
+        "Y280": ["REMIS", "ROZZI", "NAITE", "REDFN", "LEV"],
+        "Y290": ["LEV", "BLVNS", "BUNNZ", "BACCA"]
+    },
+    "sids": {},
+    "stars": {
+       "AWDAD1": {
+            "icao": "AWDAD1",
+            "name": "Awdad One",
+            "entryPoints": {
+                "LCH": ["LCH", "LFT"],
+                "LFT": ["LFT"],
+                "LSU": ["LSU"]
+            },
+            "body": ["AWDAD", "VOODO"],
+            "rwy": {
+                "KMSY2": [],
+                "KMSY11": [],
+                "KMSY20": [],
+                "KMSY29": []
+            },
+            "draw": [
+                ["LSU", "AWDAD", "LFT", "LCH"],
+                ["AWDAD", "VOODO"]
+            ]
+        },
+        "OLEDD4": {
+            "icao": "OLEDD4",
+            "name": "Oledd Four",
+            "entryPoints": {
+                "LLA": [],
+                "TBD": []
+            },
+            "body": ["LLA", "ORICH", "LUFFS", "TBD", "OLEDD"],
+            "rwy": {
+                "KMSY2": [],
+                "KMSY11": [],
+                "KMSY20": [],
+                "KMSY29": []
+            },
+            "draw": [
+                ["LLA", "ORICH", "LUFFS", "TBD", "OLEDD"]
+            ]
+        },
+        "RYTHM4": {
+            "icao": "RYTHM4",
+            "name": "Rhythm Four",
+            "entryPoints": {
+                "HEZ": ["HEZ", "ZYDCO", "FOILS"],
+                "LBY": ["LBY"],
+                "MCB": ["MCB", "ZYDCO", "FOILS"],
+                "MEI": ["MEI", "BLEUZ", "ZYDCO", "FOILS"]
+            },
+            "body": ["RYTHM", "OYSTY", "RAYOP"],
+            "rwy": {
+                "KMSY2": [],
+                "KMSY11": [],
+                "KMSY20": [],
+                "KMSY29": []
+            },
+            "draw": [
+                ["LBY", "RYTHM", "OYSTY", "RAYOP"],
+                ["MEI", "BLEUZ", "ZYDCO", "FOILS", "RYTHM"],
+                ["MCB", "ZYDCO", "HEZ"]
+            ]
+        },
+        "SLIDD2": {
+            "icao": "SLIDD2",
+            "name": "Slid Two",
+            "entryPoints": {
+                "GPT": [],
+                "SJI": []
+            },
+            "body": ["SJI", "BUGLE", "GPT", "MUDDA", "CLERY", "SLIDD"],
```

Comment:
This won't work since GPT is a possible entryPoint

Reply:
Fixed, changed to MNSTR1 arrival